doc/api.md

```diff
@@ -268,6 +268,14 @@ Some transformation plugins also allow the use of the following optional attribu
 -   **reverse**: add bidirectionality expressions to the attribute. See the **bidirectionality** transformation plugin
     in the [Data Mapping Plugins section](advanced-topics.md#bidirectionality-plugin-bidirectional) for details.
 
+Additionally for commands (which are attributes of `type` command) the following fields are optional:
+
+-   **expression** indicates that the value of the target command will not be the plain value or the command, but an
+    expression based on a combination of the reported values. See the
+    [Expression Language definition](expressionLanguage.md) for details
+-   **payloadType**: indicates how command payload will be transformed before be sent to device
```

Comment:
Maybe the enumeration of supported values should be specified? Or, if this is not related with iotagent-node-lib, then a sentence like "Please have a look to particular IOTAs documentation for allowed values of this field in each case".

Reply:
I will comment "Please have a look to particular IOTAs documentation for allowed values of this field in each case" since is related to iotagent implementation

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1209/commits/51e40ec795236d39121aedf4bf5ee538a2a7aa16